priv/repo/migrations/20220214143811_update_users_tables_age_verified.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Oli.Repo.Migrations.UpdateUsersTablesAgeVerified do
+  use Ecto.Migration
+  import Ecto.Query, warn: false
+
+  def change do
+    alter table(:users) do
+      add :age_verified, :boolean, default: true, null: false
```

Comment:
We assume that created users will have this field in true, and if you don't use this check feature, the value will default to true as well. 

Is that ok for you?

Reply:
I think we need to reverse that for potential legal reasons. If we haven't confirmed they are 13 - we cannot assume they are.

Reply:
Also means you can avoid the query in migration.